libs/di/di_constants.py

```diff
@@ -21,24 +21,6 @@
 Python file to maintain all constant msg, error, commands for data integrity tests
 """
 # DI feature control flags
-s3_range_read_flag = "S3_RANGED_READ_ENABLED"  # enable ranged reads
```

Comment:
Are these being used anywhere? Do we want to delete the file itself if all constants are removed?

Reply:
@ketanarlulkar  you should avoid taking any decision on deleting files. All DI module specific constants needs to be move or used in this files. Most of them are in params.